resultsdbupdater/consumer.py

```diff
@@ -41,8 +41,9 @@ def _consume_helper(self, msg):
         # Next, detect if the message bears the primary format we support.
         # The "FACTORY 2.0 CI UMB messages"
         # https://docs.google.com/document/d/16L5odC-B4L6iwb9dp8Ry0Xk5Sc49h9KvTHrG86fdfQM/edit
```

Comment:
Can we update this comment to reference the new spec?

Reply:
Updated to https://pagure.io/fedora-ci/messages